get-all-certs.ps1

```diff
@@ -39,4 +76,4 @@ Get-ChildItem -Recurse cert: `
     $oPem.toString() | add-content $path
   }
 
```

Comment:
and as I understand after your quick merge the script contains double code now

Reply:
Ugh, sorry about that. I did not see the diff in the merge and thought it was an empty merge.